code/modules/library/book.dm

```diff
@@ -0,0 +1,352 @@
+///Max Writeable Content Pages per book, players really don't need more than this
+#define MAX_PAGES 5
+
+/**
+  * # Standard Book
+  *
+  * Game Object which stores pages of text usually written by players, has other editable information such as the book's
+  * title, author, summary, and categories. Has other values that are generated when books are acquired through the library
+  * computer.
+  *
+  * Like other User Interfaces that heavily rely on player input, using newer tools such as TGUI presents sanitization issues
+  * so books will remain using BrowserUI until further notice.
+  */
+/obj/item/book
+	name = "book"
+	icon = 'icons/obj/library.dmi'
+	icon_state ="book"
```

Comment:
```suggestion
	icon_state = "book"
```

Reply:
forgot to include this in my last commit, will make sure this is batched with later reviews and added